S/StaticArrays/Compat.toml

```diff
@@ -1,10 +1,19 @@
 ["0-0.10"]
-julia = ["0.7", "1"]
+julia = ["0.7", "1-1.10"]
 
-["0.11-1.3"]
+["0.11"]
+julia = "1-1.10"
+
+["0.12-0"]
 julia = "1"
```

Comment:
Just to confirm, this version _is_ compatible with Julia v1.11+?

Reply:
Yes, we backported the relevant commit to v0.12.5 or so.